modules/data_warehouse/tutorial.md

```diff
@@ -0,0 +1,184 @@
+
+```
+ labels = {
+    data-warehouse = "true",
+    make-it-mine   = "true"
+  }
+```
+
+NOTE: The changes in infrastructure may lead to reduction or increase in the incurred cost.
+
+Please note: to open your recently used workspace:
+* Go to the `File` menu.
+* Select `Open Recent Workspace`.
+* Choose the desired workspace.
+
+---
+**Navigate to Data Warehouse module directory**
+
+Use the following command to change the working directory to the `modules/data_warehouse`:
+```bash
+cd modules/data_warehouse
+```
+
+**Automated deployment**
+
+Execute the below command if you want an automated deployment to happen without following the full tutorial.
+
+The step is optional and you can continue with the full tutorial if you want to understand the individual steps involved in the script.
+
+```bash
+chmod +x ./deploy.sh
+./deploy.sh
+```
+
+
+## Gather the required information for intializing gcloud command
+
+In this step you will gather the information required for the deployment of the solution
+
+---
+**Project ID**
+
+Use the following command to see the projectId:
+
+```bash
+gcloud config get project
+```
+
+```
+Use above output to set the <var>PROJECT_ID</var>
+```
+---
+**Deployment Region**
+
+```
+Provide the region (e.g. us-central1) where the top level deployment resources were created for the deployment <var>REGION</var>
+```
+---
+**Deployment Name**
+
+Run the following command to get the existing deployment name:
+```bash
+gcloud infra-manager deployments list --location <var>REGION</var> --filter="labels.goog-solutions-console-deployment-name:* AND labels.goog-solutions-console-solution-id:data-warehouse"
+```
+
+```
+Use the NAME value of the above output to set the <var>DEPLOYMENT_NAME</var>
+```
+
+
+## Deploy the solution
+
+
+---
+**Fetch Deployment details**
+```bash
+gcloud infra-manager deployments describe <var>DEPLOYMENT_NAME</var> --location <var>REGION</var>
+```
+From the output of this command, note down the input values provided in the existing deployment in the `terraformBlueprint.inputValues` section.
+
+Also note the serviceAccount from the output of this command. The value of this field is of the form 
+```
+projects/<var>PROJECT_ID</var>/serviceAccounts/<serice-account>@<var>PROJECT_ID</var>.iam.gserviceaccount.com
+```
+
+```
+Note <service-account> part and set the <var>SERVICE_ACCOUNT</var> value.
+You can also set it to any exising service account.
+```
+
+---
+**Assign the required roles to the service account**
+```bash
+while IFS= read -r role || [[ -n "$role" ]]
+do \
+gcloud projects add-iam-policy-binding <var>PROJECT_ID</var> \
+  --member="serviceAccount:<var>SERVICE_ACCOUNT</var>@<var>PROJECT_ID</var>.iam.gserviceaccount.com" \
+  --role="$role"
+done < "roles.txt"
+```
+
+---
+**Create Terraform input file**
+
+Create a file named `input.tfvars`.
+
+Find the sample content below and modify it by providing the respective details.
+```
+delete_contents_on_destroy=True
```

Comment:
This input is not required for data-warehouse: https://github.com/GoogleCloudPlatform/solutionbuilder-terraform-google-bigquery/blob/main/modules/data_warehouse/variables.tf